src/test/java/no/unit/nva/doi/FetchDoiMetadataTest.java

```diff
@@ -33,43 +41,54 @@
     public static final String ERROR_KEY = "error";
     public static final String DATACITE_RESPONSE = "";
 
+    public static final String VALID_DOI_WITH_DOI_PREFIX = "doi:10.123.4.5/124";
+    public static final String VALID_DOI_WITHOUT_DOI_PREFIX = "10.123.4.5/124";
+
+    private static final Context mockLambdaContext = createMockContext();
+    private static final CrossRefClient crossRefClient = createCrossRefClient();
+
     public static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
 
     @Test
     public void successfulResponse() throws Exception {
         DataciteClient dataciteClient = Mockito.mock(DataciteClient.class);
```

Comment:
Can be moved to class level and annotated with @Mock

Reply:
To my understanding the `@Mock` tag replaces the `mock` command. The methods have some `when` in the setups. I am not sure I can have the tag and keep the methods as they are now. If I keep the tags, then I would have to create an `@BeforeAll` method to initialize them, which becomes more complex than field initialization.